core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java

```diff
@@ -33,9 +34,16 @@
  */
 public class CamelModelFactory {
 	
-	private static final String CAMEL_VERSION_2_15 = "2.15";
-	private static final String FUSE_VERSION_6_3_0_REDHAT_077 = "6.3.0.redhat-077";
-	private static final String FUSE_VERSION_6_2_1_REDHAT_084 = "6.2.1.redhat-084";
+	private static final Map<String, String> camelVersionToFuseBOMMapping;
+	static {
+		camelVersionToFuseBOMMapping = new HashMap<>();
+		camelVersionToFuseBOMMapping.put("2.15.1.redhat-621084", "6.2.1.redhat-084");
+		camelVersionToFuseBOMMapping.put("2.15.1.redhat-621117", "6.2.1.redhat-117");
+		camelVersionToFuseBOMMapping.put("2.17.0.redhat-630175", "6.3.0.redhat-175");
+		camelVersionToFuseBOMMapping.put("2.17.3", 				 "6.3.0.redhat-175");
```

Comment:
only here at github or also locally in your eclipse source code? I aligned the second param with the ones above.